tools-local/tasks/mobile.tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -58,6 +58,10 @@ public class MonoAOTCompiler : Microsoft.Build.Utilities.Task
     /// </summary>
     public bool UseAotDataFile { get; set; } = true;
 
+    public string? AotProfilePath { get; set; }
+
+    public ITaskItem[]? Profilers { get; set; }
```

Comment:
You can change the type to `string[]`. `ITaskItem` is useful if you want to access the metadata on the items, which is not the case here.